fabfile/pgbench_confs/pgbench_custom.ini

```diff
@@ -1,5 +1,5 @@
 [DEFAULT]
-postgres_citus_versions: [('12.1', 'release-9.1'), ('12.1', 'release-9.0')]
+postgres_citus_versions: [('12.1', 'enterprise-master'), ('12.1', 'release-9.1')]
```

Comment:
If it is hard let's skip, but is it possible to fetch the latest release tag and put automatically -- unless a specific value is specified?

For example, can we add a special keyword, like `latest-version` and have
``` 
postgres_citus_versions: [('12.1', 'enterprise-master'), ('12.1', 'latest-version')]
```

That said, if it is going to take more than 1 hour to do that, we can skip

Reply:
I guess in fab scripts we can check if it is 'latest-version` if so, we can have a function that returns the latest release we have. I will check if it is easy to do.

Reply:
I guess this will take more than 1 hour to do but it is a nice thing to have, so we might do it in another PR :)